### ISSUE TITLE
Decoder's encoder_attention_mask input should be called decoder_attention_mask

### ISSUE THREAD

**Issue by praeclarum** [Ki6an/fastT5#59]
The exported decoder feed name says `encoder_attention_mask` but I think it should be called `decoder_attention_mask`. The same applies for the init decoder.

https://github.com/Ki6an/fastT5/blob/20441b33394e71f7612f39f228ecbe1925cd10ae/fastT5/onnx_exporter.py#L215

The behavior is correct, I was just confused on how to set the mask correctly for the decoder.

**Comment by praeclarum** [comment]
Looks like it really is the encoder attention mask? At least the network fails when I try to use the decoder's.
